src/components/Admin/ListUser/ListUser.js

```diff
@@ -184,6 +188,56 @@ export default class ListUser extends Component {
     });
   };
 
+  handleSearch = value => {
+    this.setState({
+      search: true,
+    });
+    let url;
```

Comment:
```
let url = `...`
```

If you're using template literals then you won't need `+` to combine urls/params

Reply:
Fixed it.